homeassistant/components/blink/binary_sensor.py

```diff
@@ -32,10 +26,6 @@
         device_class=BinarySensorDeviceClass.BATTERY,
         entity_category=EntityCategory.DIAGNOSTIC,
     ),
-    BinarySensorEntityDescription(
```

Comment:
2 things:
1. If I read your breaking change, I expected the one under this to be removed.
2. We should give users 6 months to update their automations. So if you announce removal now (and add a comment to the sensor so we know it should be removed) we can remove it in June somewhere

Reply:
It's also a bit confusing I suppose.  Camera armed is actually arming motion detection for the camera.  The  other sensor is indicating if motion is detected (line 39 - )  I changed the name to motion detection for the switch as I thought it made more sense, but maybe not?

Reply:
Like, if technically possible we would like to give a 6 month depreciation period for removing entities, so people have time to update automations and stuff